forge.config.ts

```diff
@@ -21,7 +21,15 @@ if (process.env['WINDOWS_CODESIGN_FILE']) {
 
   if (certExists) {
     process.env['WINDOWS_CODESIGN_FILE'] = certPath;
+  } else if (process.env['SIGN_OR_DIE']) {
+    throw new Error('Did not find Windows codesign file');
   }
+
+  if (!process.env['WINDOWS_CODESIGN_PASSWORD'] && process.env['SIGN_OR_DIE']) {
+    throw new Error('Did not find "WINDOWS_CODESIGN_PASSWORD" env variable');
+  }
+} else if (process.env['SIGN_OR_DIE']) {
+  throw new Error('Did not find "WINDOWS_CODESIGN_FILE" env variable');
```

Comment:
Do we need this check if we're checking for the lack of WINDOWS_CODESIGN_PASSWORD in the line above? Could we just check for the presence of WINDOWS_CODESIGN_PASSWORD and WINDOWS_CODESIGN_FILE, and throw if either aren't present/are an empty string?

Reply:
Tweaked the logic in 91e0fb84e3864dab47294ea77bf342adcb26902f, if that's good with you then merge away. :shipit: